### DISCUSSION TITLE
Connection error dbeaver docker postgres

### DISCUSSION THREAD

**Discussion by austiamel47** [dbeaver/dbeaver#11051] [Q&A]
Hey guys! I don't know why I am having this error to connect on my docker with Postgres. Is there another way to test this connection without Dbeaver? (noob here) hehe thanks!

**Comment by JohnKarpenter** [comment]
I couldn't find an answer to my question when searching online or in the dbeaver documentation so I'm hoping someone here can help me.

Basically what happens is when I'm writing a query in the editor, once I run it the results tab pops up. This is good. But once I edit the query and re-run the code, the previous results are replaced in the results tab by the results of the updated code.

How can I keep the results of previous queries shown after running new queries in the same tab so that I can compare differences more easily?https://trackeasy.fun/usps/

Thanks
